cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -2195,6 +2223,14 @@
 (rule (flt $F32 a b) (fpu_rrr (FpuOPRRR.FltS) $I64 a b))
 (rule (flt $F64 a b) (fpu_rrr (FpuOPRRR.FltD) $I64 a b))
 
+(decl fmin_ (Type Reg Reg) Reg)
```

Comment:
Looks like we can't name this `fmin` since it conflicts with the preexisting declaration for the `fmin` clif instruction. I guess adding a prefix to all RISCV instruction helpers would make sense? The x86 backend has `x64_*` so we could add something like `rv_*`, what do you guys think about this?

If everyone agrees I don't mind doing a pass on the backend adding the helpers and making sure everything is aligned.